<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>YangJanus</class>
 <widget class="QDialog" name="YangJanus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>281</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget, QLabel{
                        color:white;
                        border: 1px solid white;
                        padding:0;
                        border-radius:5px;
							background-color:rgb(20,50,70);
                      /**  background-color:rgba(250,170,0,150);**/
  }


QTabWidget{

border: 1px solid white;
background-color:rgb(20,50,70);
}
QTabWidget::tab-bar{
alignment:left;
left:1px;
}
QTabWidget::pane { 
/*background-color:rgb(20,50,70);*/
border:1px solid white;
} 
QTabBar{

background-color:white;
}
QTabBar::tab{
min-height:28px;
padding:0 10px;
border:0;
margin:1px 1px 0 0;
background-color:rgb(20,50,70);
}
QTabBar::tab:first{
margin-left:1px;
}
QTabBar::tab:hover{
color:cyan;
}
QTabBar::tab:selected{
background-color: rgb(20, 100, 150);
}
QTabBar::tab:selected:hover{
}

QTabBar::tear{
}
QTabBar::scroller{
}
QMenu {
     background-color: rgb(20,50,70); 
     border: 1px solid white;
 }
  
 QMenu::item {

     background-color: transparent;
     padding:8px 32px;
     margin:0px 8px;
     border-bottom:1px solid #DBDBDB;
 }
  
 QMenu::item:selected { 
     background-color: #2dabf9;
 }</string>
  </property>
  <widget class="QPushButton" name="m_b_rec">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>createUser</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_l_url">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>URL:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_url">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>601</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>http://10.42.0.1:7080/whip/create</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_l_url_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Data:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_data">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>601</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>{&quot;id&quot;: &quot;metaRTC&quot;, &quot;room&quot;: 1234}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_b_rec_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>210</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_retInfo">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>601</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
